Formula/w/wxmaxima.rb

```diff
@@ -72,6 +72,7 @@ def caveats
     # Error: Unable to initialize GTK+, is DISPLAY set properly
     return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
 
-    assert_match "algebra", shell_output("#{bin}/wxmaxima --help 2>&1")
+    assert_equal "wxMaxima #{version}", shell_output(bin/"wxmaxima --version 2>&1").chomp
+    assert_match "extra Maxima arguments", shell_output("#{bin}/wxmaxima --help 2>&1 || true")
```

Comment:
Does `wxmaxima --help` not exit with `0`? If so you can pass the expected exit code as the second argument of `shell_output`.

Reply:
It doesn't, surprisingly. Thanks for the comment, I suspected there might be a better way to do this :)